### PR TITLE
SDS Hackathon Experience

### PR DESCRIPTION
**Pull request by samagra14**
This aim of this pull request is the submission of a blog post sharing the experience of SDS hackathon along with minor changes in the readme file so as to correct the headings.

### CODE REVIEW

File: _posts/2017/sds_hackathon_2017.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+### The development process
+
+>The main motive of the hackathon was to make freshies aware of the various domains of the technology and to make them comfortable with the process of software development.
+
+Each of our team member was on a seperate ground as far as Android Development was concerned. So making them work together (Being careful of the fact that the first yearites do  not lose their interest owing to the intimidating nature of the project) at the same level was a near Herculean task.
+
+We began by dividing the tasks by carefully analysing the prior knowledge of a particular member and their capacity to learn new stuff in the given time frame and assigning respective tasks to them.
+
+The constant help from the members of MDG and SDSLabs proved to be a great enthu booster. There were times when we felt that our idea was infeasible and that we would not be able to complete the project. But the perks of winning and the thought of achieving something which could be of great potential was sufficient enogh to raise our morales and to put our asses to work. Finally, the best part was __infinite food__ and drinks which were provided by the organisers.
```
Comment (csoni111) [inline _posts/2017/sds_hackathon_2017.md]: ~enogh~ enough

**Comment by arpit270896** [comment]
An image or demo video would be great. :slightly_smiling_face:

File: _posts/2017/sds_hackathon_2017.md (hunk 1 above)
Comment (samagra14) [reply _posts/2017/sds_hackathon_2017.md]: Checked .

**Comment by samagra14** [comment]
I have added a few images but not videos because video looked quite inelegant as the textblock changes very rapidly even at the slightest distortion of the mobile device.

**Pull request merged**